.vitepress/docs/components/SchemaPlayground.vue

```diff
@@ -10,9 +10,14 @@
     </div>
     <div>
       <SchemaForm
+        @submit.prevent="onSubmit"
```

Comment:
I think we can remove the `.prevent` here as it’s not necessary, right?

Reply:
It actually is, when not present the page reloads itself

Reply:
Oh I see! That makes sense! I think we need to add that listener with `.prevent` to that root `<form>` element in SchemaForm.vue.